### PR TITLE
#BE-2803 Okta SAML Documentation Update

### PR DESCRIPTION
**Pull request by atkncamurcu**

### CODE REVIEW

**Comment by burakberkkeskin** [comment]
- The document page overviews are missing. What is the document page is describing, what will happend when the follow the page?

- There is no overview for [LDAP](https://deploy-preview-311--reverent-galileo-8ef035.netlify.app/account/ldap-login).

- There is no overview for [SSO Login Dist & Store](https://deploy-preview-311--reverent-galileo-8ef035.netlify.app/account/sso-login/single-sign-on-login)

**Comment by burakberkkeskin** [comment]
- I added some cautions about self-hosted and some directives.
- The okta configuration for the Appcircle dashboard login should also be edited as in okta dist and store page.

**Comment by burakberkkeskin** [comment]
- After the Dist and Store SSO configuration is done, there is no detailed description or page about how to enable SSO for dist and store profiles in the [page](https://deploy-preview-311--reverent-galileo-8ef035.netlify.app/account/sso-login/okta-saml/).

File: docs/account/sso-login/okta-saml.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+- Go back to Appcircle, upload this XML file by clicking the button under **Import SAML Configuration**
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2777-sso-saml1-new.png' />
+
+- Check all the settings on this page and confirm that Redirect and SSO URLs are imported correctly. You can check if the X509 Certificate is imported correctly as well. If you want to enter multiple certificates you can separate them by using a comma between them. Please be aware that you need to remove any new lines or file headers from this edit box. This edit box only accepts a long base64 encoded string.
+
+- Hit `Save` button and save the SAML configuration on Appcircle.
+
+- You can enable the SSO Login in Enterprise App Store Settings
```
Comment (burakberkkeskin) [inline docs/account/sso-login/okta-saml.md:96]: After the SSO integration is done, there is no detailed description or page about how to enable SSO for dist and store profiles in the page.

File: docs/account/sso/okta-saml.md
Comment (burakberkkeskin) [inline docs/account/sso/okta-saml.md:1]: - The document page overviews are missing. What is the document page describing? What will happen when someone follows the page?
- I added some cautions about self-hosted and some directives.
- The okta configuration for the Appcircle dashboard login should also be edited, as in the Okta dist and store page.

File: docs/account/ldap-login.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+title: Enterprise Appstore and Testing Distribution LDAP Login
+metaTitle: Enterprise Appstore and Testing Distribution LDAP Login
+metaDescription: Enterprise Appstore and Testing Distribution LDAP Login
+sidebar_position: 14
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# LDAP Login
+
```
Comment (burakberkkeskin) [inline docs/account/ldap-login.md:11]: The document page overviews are missing. What is the document page describing? What will happen when someone follows the page?

File: docs/account/sso-login/okta-saml.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+title: Okta SAML
+metaTitle: Okta SAML
+metaDescription: Okta SAML
+sidebar_position: 3
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Okta SAML
+
```
Comment (burakberkkeskin) [inline docs/account/sso-login/okta-saml.md:11]: The document page overviews are missing. What is the document page describing? What will happen when someone follows the page?

**Review by burakberkkeskin — CHANGES_REQUESTED** [review]

**Review by csonuryilmaz — APPROVED** [review]

**Pull request merged**